packages/prompts/src/limit-options.ts

```diff
@@ -8,12 +8,18 @@ export interface LimitOptionsParams<TOption> extends CommonOptions {
 	maxItems: number | undefined;
 	cursor: number;
 	style: (option: TOption, active: boolean) => string;
+	/**
+	 * Custom format for overflow indicators.
+	 * Defaults to '    ...' if not provided.
+	 */
+	overflowFormat?: string;
 }
 
 export const limitOptions = <TOption>(params: LimitOptionsParams<TOption>): string[] => {
 	const { cursor, options, style } = params;
 	const output: Writable = params.output ?? process.stdout;
 	const rows = output instanceof WriteStream && output.rows !== undefined ? output.rows : 10;
+	const overflowFormat = params.overflowFormat ?? color.dim('    ...');
```

Comment:
did we ever use this? it looks like the only places we set it, we use the default

Reply:
looking at the snapshots, it seems this changes how some of the existing prompts render
so maybe we should just revert this bit as it seems unnecessary for the autocomplete

Reply:
should we maybe revert this @dreyfus92?

unless you can figure out why it exists

Reply:
the multi-select prompt had no spaces before, and now has two, because of this

that's what i want to understand. does it look better with the two spaces? why were they added?